tests/queries/0_stateless/02868_gcd_codec_test_data.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE IF EXISTS table_none;
```

Comment:
For coverage, please add basic tests (`CREATE TABLE`, `INSERT`, `SELECT`) for all supported data types (`[U]Int{8,16,32,64,128,256`, decimal types, all supported date time types). I it would also be interesting to check that no bad things happen if negative values are used with unsigned data types.

Reply:
Oh, and what happens if only 0 (zero) values are inserted?